cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -37,5 +126,65 @@ def from_domain_expert_metadata(
         return cls(**metadata_as_dict)
 
 
-class InformationArchitectRules(RuleModel):
-    metadata: InformationArchitectMetadata
+class InformationClass(DomainClass):
+    """
+    Class is a category of things that share a common set of attributes and relationships.
+
+    Args:
+        class_: The class ID of the class.
+        name: The name of the class.
+        parent: The parent class of the class.
+        source: Source of information for given resource
+        match_type: The match type of the resource being described and the source entity.
+    """
+
+    source: Namespace | None = None
+    match_type: MatchType | None = None
+
+    @field_validator("source", mode="before")
+    def fix_namespace_ending(cls, value):
+        if value:
+            return Namespace(TypeAdapter(HttpUrl).validate_python(value))
+        return value
+
+
+class InformationProperty(DomainProperty):
+    """
+    A property is a characteristic of a class. It is a named attribute of a class that describes a range of values
+    or a relationship to another class.
+
+    Args:
+        class_: Class ID to which property belongs
+        property: Property ID of the property
+        name: Property name.
```

Comment:
I haven't seen this is the sheets we have.
```suggestion
```

Reply:
There is, name is used by a customer who has cryptic classes (class -> class_id) where they need human readable name , thus we have name added